server/utils/utils.go

```diff
@@ -3,10 +3,34 @@ package utils
 import (
 	"crypto/sha1"
 	"encoding/hex"
+	"fmt"
 
+	"github.com/dgrijalva/jwt-go"
 	"github.com/xdg-go/pbkdf2"
 )
 
+type CustomClaims struct {
+	UserAgent       string `json:"user_agent"`
+	SecChUaPlatform string `json:"sec-ch-ua-platform"`
+	SecChUaMobile   string `json:"sec-ch-ua-mobile"`
+	SecFetchSite    string `json:"sec-fetch-site"`
+	SecFetchDest    string `json:"sec-fetch-dest"`
+	SecFetchMode    string `json:"sec-fetch-mode"`
+	Referer         string `json:"referer"`
+	Origin          string `json:"origin"`
+	ContentLength   string `json:"content-length"`
+	jwt.StandardClaims
+}
+
+func GenerateToken(secret string, claims *CustomClaims) (string, error) {
```

Comment:
I think this function is not used.

Reply:
Yes, this code is deprecated, will remove that.